src/cryptoadvance/specter/templates/includes/recipient-box.html

```diff
@@ -488,7 +494,12 @@
 		markRecipient(is_address_mine) {
```

Comment:
@otkstdio this is the relevant part that changes stuff in the UI if the address belongs to the wallet of the sender